<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LP2-AP-52100881" failures="1" tests="1" name="EdgeTest" time="12.202" errors="0" timestamp="2024-11-04T14:45:48 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.182">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:55)
at junit.framework.Assert.assertTrue(Assert.java:22)
at junit.framework.Assert.assertTrue(Assert.java:31)
at com.pageclasses.myntraNewValidationPage.testing(myntraNewValidationPage.java:47)
at com.stepdef.myntraNewValidationstepDef.myntraLoginPage(myntraNewValidationstepDef.java:31)
at âœ½.I navigate to myntra loginpage newPage(file:///C:/testing_RTP/my-app/src/test/resources/features/Featurefile.feature:80)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- EdgeTest -->
